chain/network/src/ibf_peer_set.rs

```diff
@@ -0,0 +1,231 @@
+use near_primitives::network::PeerId;
+use std::collections::HashMap;
+use std::sync::{Arc, Mutex};
+
+use crate::ibf_set::IbfSet;
+use crate::routing::Edge;
+
+pub type SlotMapId = u64;
+
+#[derive(Default)]
+struct SlotMap {
+    id: u64,
+    id2e: HashMap<SlotMapId, SimpleEdge>,
+    e2id: HashMap<SimpleEdge, SlotMapId>,
+}
+
+#[allow(dead_code)]
```

Comment:
Why do we need this? Also it is better to place it on individual functions so that it is easier to see which ones are not used

Reply:
That's not needed anymore. It was needed when I was writing the code, and those function were not used yet.